### ISSUE TITLE
Target size is not the same as input size

### ISSUE THREAD

**Issue by gengxing0824** [dxli94/WLASL#18]
！loc_loss = F.binary_cross_entropy_with_logits(per_frame_logits, labels)
per_frame_logits.shape = (1, 100, 75)
labels.shape = (1)
per_frame_logits.shape != labels.shape
How to solve this problem？
